.gitignore

```diff
@@ -31,6 +31,8 @@ components/versions/versions.json
 .helm-chart-release
 *.tgz
 
+components/public-api/java/bin
```

Comment:
Feel free to remove and rebase, it's been added in https://github.com/gitpod-io/gitpod/pull/19887 to the respective folder's gitignore.